TAScheduler/AcceptanceTests/coursecreationtests.py

```diff
@@ -1,12 +1,18 @@
 from django.db.models import QuerySet
 from django.test import TestCase
 from django.test import Client
-from TAScheduler.models import Course, Section
-from django.contrib.auth.models import User
+from TAScheduler.models import Course, Section, Profile
+from django.contrib.auth.models import User, Group
+
 
 class TestCourseCreationGet(TestCase):
 
     def setUp(self):
+        self.ta_group, created = Group.objects.get_or_create(name='ta')
+
+        self.instructor_group, created=Group.objects.get_or_create(name='instructor')
+        self.admin_group, created=Group.objects.get_or_create(name="manager")
```

Comment:
Instead of manually creating these groups during the test setup, you might want to just call `TASchedulerAppConfig.ready(None)`, which is the application startup procedure that creates these automatically during normal operation. Check out `unittests_coursemanagement.py` for an example.

Reply:
Cool, I would update this, but my github decided to break. I will try and fix it and update this.